### DISCUSSION TITLE
Error when deploying Git repository in Vercel with Github account.

### DISCUSSION THREAD

**Discussion by 427green** [vercel/community#5033] [Help]
### Summary

Hi, I tried to deploy in Vercel using my github account but got the following error. It has been going on for days, how can I help?
![image](https://github.com/vercel/community/assets/78639912/802c7344-b170-4bea-95af-237cab56a6f1)

Is this temporary? I searched and found someone in the past with a similar problem, but could not find a solution.

### Steps to Reproduce

https://vercel.com/new/clone Deploy in your github account at
![image](https://github.com/vercel/community/assets/78639912/ad741d14-628d-4c2a-9982-082ea4a98a9a)
It also happens in GitLab

**Comment by amyegan** [comment]
If you remove the GitHub connection in your account and add it again, that will reset the connection. You'll need to make sure you give the integration appropriate permission to work with future repos. Please try that and let me know if it works for you 🙏
